README.md

```diff
@@ -16,6 +16,12 @@ This WordPress plugin allows you to **use Discourse as a community engine for yo
 
 ## Installation
 
+### Plugin manager from your `wp-admin`
+
+Download the [zip](https://github.com/discourse/wp-discourse/archive/master.zip) and upload it to your wordpress instance from `wp-admin`/`plugins`/`Add new`/`Upload Plugin`
```

Comment:
should recommend downloading from the github releases page and not the master zip file